config/public-api-v1.yml

```diff
@@ -31,31 +31,365 @@ paths:
+            museums and other sites for learning, which are recognised as an
+            important resource for engaging students in cultural and social
+            issues.Through seminars and studio-based activities, students will
+            study the concepts, processes and skills of art, craft and design,
+            sharing their knowledge and understanding with other student
+            teachers and consider how it relates to the secondary curriculum.
+            Towards the end of the PGCE, students will build on their own
+            practice by initiating a curriculum development project, culminating
+            in the display of their own work in a final exhibition that
+            represents their personal philosophy for art and design education.
+        accredited_body_code:
+          type: string
+          description: |
+            Unique provider-code for the accredited body of this course. Only
+            present if the course is accredited by a provider other than the one
+            running this course.
+          maxLength: 3
+          example: 2FR
+        age_range_in_years:
+          type: string
+          description: |
+            Description of the age-range of the pupils that a course is specified for.
+          example: 11_to_14
+        applications_open_from:
+          type: string
+          format: date
+          description: "Date from which applications can be submitted."
+          example: "2019-10-08"
+        bursary_amount:
+          type: integer
+          description: "Bursary amount for this course."
+          example: 9000
+        bursary_requirements:
+          type: string
+          description: "Description of requirements to be eligible for a bursary."
+          example: "a degree of 2:2 or above in any subject"
+        changed_at:
+          type: string
+          format: date-time
+          description: |
+            Date-time timestamp of when this course or any of it's related data changed.
+          example: "2019-06-13T10:44:31Z"
         code:
           type: string
+          description: |
+            Code that uniquely identifies this course within it's providers list of courses.
+          example: 3GTY
+          maxLength: 4
+        course_length:
+          type: string
+          description: |
+            Text describing how long the course runs.
+          example: "OneYear"
+        created_at:
+          type: string
+          format: date-time
+          description: "Timestamp of when this course was created."
+          example: "2019-06-13T10:44:31Z"
+        fee_details:
+          type: string
+          format: markdown
+          description: "Further details about the fees for this course, if applicable."
+          example: fee_details
+        fee_international:
+          type: integer
+          description: "Fee for international students (optional)."
+          example: 13000
+        fee_uk_eu:
```

Comment:
Should we be baking `eu` in here? Not sure what the alternative would be `fee_resident` or something like that?

Reply:
Good call. How about `fee_domestic`.

Reply:
Not sure. Is that a product/policy question?

Reply:
I'll ask PaulG to have a look at these.